packages/material-ui/src/Chip/Chip.js

```diff
@@ -87,7 +87,7 @@ export const styles = (theme) => {
       userSelect: 'none',
       WebkitTapHighlightColor: 'transparent',
       cursor: 'pointer',
-      '&:hover, &:focus': {
+      '&$focusVisible, &:hover': {
         backgroundColor: emphasize(backgroundColor, 0.08),
```

Comment:
Related note: we should probably look into applying https://material.io/design/interaction/states.html. This is would be for another pull request.

Reply:
Are you suggesting we apply different styling to focus compared with hover?

Reply:
Yes, it's something we have started to do, see #10870 for progress.